integration_tests/e2e/addAlertPage.cy.ts

```diff
@@ -93,7 +93,6 @@ context('Add Alert Page', () => {
       cy.setupUserAuth({
         roles: [Role.GlobalSearch],
         caseLoads: [{ caseloadFunction: '', caseLoadId: 'ZZZ', currentlyActive: true, description: '', type: '' }],
-        activeCaseLoadId: 'ZZZ',
```

Comment:
Don't need this now, derived from the caseloads list above for which is `currentlyActive`